torch/_functorch/_aot_autograd/dispatch_and_compile_graph.py

```diff
@@ -210,11 +210,21 @@ def _map_assigned_buffer_to_proxy(_mod, name, buffer):
                 colored=True,
             ),
         )
+
+        def fw_metadata_str():
+            return "\n\n".join(
+                [
+                    f"fw_metadata: {str(fw_metadata)}",
+                    f"maybe_subclass_meta: {str(maybe_subclass_meta)}",
+                    fw_module.print_readable(
+                        print_output=False, include_stride=True, include_device=True
+                    ),
+                ]
+            )
+
         trace_structured(
             "aot_forward_graph",
-            payload_fn=lambda: fw_module.print_readable(
-                print_output=False, include_stride=True, include_device=True
-            ),
+            payload_fn=lambda: fw_metadata_str,
```

Comment:
This doesn't do what you want, payload_fn=fw_metadata_str

Have you inspected the output tlparse yet?

Reply:
Oh it looks like you didn't exercise this one

Reply:
whoops... thank you